### PR TITLE
Prettier JS fixes

### PR DESCRIPTION
**Pull request by ntwb**
As noted at https://github.com/stylelint/stylelint/pull/2834#pullrequestreview-58393680 there was a strange _prettier_ change in that PR that was unexpected, this PR is to to test against `master` any missing changes that should have been added by Prettier already.

• I've added `prettier:check` and `prettier:fix` npm commands, `npm run prettier:check` and `npm run prettier:fix` respectively to help test changes in this PR

Running `npm run prettier:check` results in:
```shell
prettier '**/*.js'  --list-different

.coverage/lcov-report/prettify.js
.coverage/lcov-report/sorter.js
decls/flow-typed/lodash.js

decls/flow-typed/lodash.js: SyntaxError: default is a reserved word (463:55)
  461 |     conforms(source: Object): Function;
  462 |     constant<T>(value: T): () => T;
> 463 |     defaultTo<T1:string|boolean|Object,T2>(value: T1, default: T2): T1;
      |                                                       ^
  464 |     // NaN is a number instead of its own type, otherwise it would behave like null/void
  465 |     defaultTo<T1:number,T2>(value: T1, default: T2): T1|T2;
  466 |     defaultTo<T1:void|null,T2>(value: T1, default: T2): T2;
decls/postcss.js
decls/stylelint.js
lib/__tests__/standalone.test.js
lib/getConfigForFile.js
lib/normalizeRuleSettings.js
lib/postcssPlugin.js
lib/rules/indentation/__tests__/functions.js
lib/rules/indentation/index.js
lib/standalone.js
```

The results after running `npm run prettier:fix` are the file changes in this PR (apart from `package.json` change adding the two script commands)

- [ ] decls/postcss.js
- [ ] decls/stylelint.js
- [ ] lib/__tests__/standalone.test.js
- [ ] lib/getConfigForFile.js
- [ ] lib/normalizeRuleSettings.js
- [ ] lib/postcssPlugin.js
- [ ] lib/rules/indentation/__tests__/functions.js
- [ ] lib/rules/indentation/index.js
- [ ] lib/standalone.js

@stylelint/core could you review this PR please and we'll get this shipped in 8.1.0

### CODE REVIEW

File: lib/postcssPlugin.js [hunk 1]
```diff
@@ -98,16 +98,23 @@ const postcss/*: postcssType*/ = require("postcss")
...
-  const stylelint/*: stylelint$internalApi*/ = createStylelint(tailoredOptions)
+    : options;
+  const stylelint /*: stylelint$internalApi*/ = createStylelint(
+    tailoredOptions
+  );
 
-  return (root/*: rootParamT*/, result/*: resultParamT*/)/*: Promise<any>*/ => {
-    let filePath = options.from || _.get(root, "source.input.file")
+  return (
+    root /*: rootParamT*/,
```
Comment (ntwb) [inline lib/postcssPlugin.js]: The trailing comma here at the end of line causes a flow error:

```shell
Error: lib/postcssPlugin.js:114
114:     root /*: rootParamT*/,
                              ^ Unexpected token ,
```

**Comment by ntwb** [comment]
The `decls/flow-typed/lodash.js` resulting `SyntaxError` above after the initial `npm run prettier:check` check is no longer an issue after commit https://github.com/stylelint/stylelint/pull/2843/commits/7fcf43165f2eefa7e5987fe5a2c54994be4a8f77 in #2843 is merged 😄

File: .travis.yml [hunk 2]
```diff
@@ -33,7 +33,7 @@ matrix:
       sudo: required
 
 before_install:
-  - npm install -g npm@latest
+  - npm install -g npm@5.3.0
```
Comment (ntwb) [inline .travis.yml]: @jeddy3 Could you explain why you made this change?

FYI: `npm install -g npm@latest` is valid and the recommended command to [update npm](https://docs.npmjs.com/all#updating-npm)

p.s. The current npm release is `5.4.0`

Comment (jeddy3) [reply .travis.yml]: Yep, sure. I was trying to get this branch to pass by pinning the npm version to 5.3.0 as there seems to be [an issue](https://github.com/facebook/flow/issues/4733) with flow and 5.4.0. Unfortunately, I ran out of time to finish this up. Feel free to revert or pick this up.

Comment (ntwb) [reply .travis.yml]: Thanks, makes sense, happy to keep npm pinned at 5.3.0 until upstream issue is resolved

Comment (ntwb) [reply .travis.yml]: As this is causing issues with contributors pull requests I've created #2849 and merged this change into `master` via commit https://github.com/stylelint/stylelint/commit/5d75094285bea5a6cc4c218a61c6a7d48244e2e7 just now.

Comment (jeddy3) [reply .travis.yml]: @ntwb Thanks!

**Pull request merged**